woocommerce/woocommerce-android
# [Activate] Remove the magic link experiment request experiment

This PR removes the magic link experiment request experiment after it's been concluded that the `enhanced` variant (which shows the button to request a magic link on the password screen) performs the best.

![image](https://user-images.githubusercontent.com/1522856/194077575-57c2e248-368a-4c41-9655-3dca7046a194.png)

**To test:**
1. Start the app
2. Log out if you're logged in
3. Enter your non-A8C email address
4. Verify the password screen contains the magic link button

**Note:** Please wait until #7477 is merged before merging this PR.
